com.adilhanney.ricochlime.json

```diff
@@ -0,0 +1,85 @@
+{
+	"app-id": "com.adilhanney.ricochlime",
+	"runtime": "org.freedesktop.Platform",
+	"runtime-version": "22.08",
+	"sdk": "org.freedesktop.Sdk",
+	"command": "ricochlime",
+	"separate-locales": false,
+	"finish-args": [
+		"--share=network",
+		"--share=ipc",
+		"--socket=fallback-x11",
+		"--socket=wayland",
+		"--device=dri",
+		"--socket=pulseaudio"
+	],
+	"modules": [
+		{
+			"name": "ricochlime",
+			"buildsystem": "simple",
+			"only-arches": [
+				"x86_64"
+			],
+			"build-commands": [
+				"mkdir -p Ricochlime",
+				"tar -xf Ricochlime_*.tar.gz -C Ricochlime",
+				"cp -r Ricochlime /app/",
+				"chmod +x /app/Ricochlime/ricochlime",
+				"mkdir -p /app/bin",
+				"mkdir -p /app/lib",
+				"ln -s /app/Ricochlime/ricochlime /app/bin/ricochlime",
+				"ln -s /app/Ricochlime/lib/* /app/lib/",
+				"install -Dm644 com.adilhanney.ricochlime.metainfo.xml -t /app/share/metainfo/",
+				"install -Dm644 icon-512x512.png /app/share/icons/hicolor/512x512/apps/com.adilhanney.ricochlime.png",
+				"install -Dm644 com.adilhanney.ricochlime.desktop -t /app/share/applications/"
+			],
+			"sources": [
+				{
+					"type": "file",
+					"url": "https://raw.githubusercontent.com/adil192/ricochlime/v0.4.2/flatpak/com.adilhanney.ricochlime.metainfo.xml",
```

Comment:
any reason why these files are not in the tarball?

Reply:
The `tar.gz` file is just the output of the flutter build command, which I use for both Flatpak and AppImage. https://github.com/flathub/flathub/pull/3459#discussion_r963122325

Reply:
and they could be used by both the same way. A Linux app without a .desktop or an icon, it is a bug. The appstream file is extra bonus (that flatpak require)

So it should be part of the build artifacts.